packages/server/package.json

```diff
@@ -9,11 +9,14 @@
   "author": "Raha",
   "license": "Apache-2.0",
   "private": true,
+  "engines": {
+    "node": "9.x"
```

Comment:
Please use correct syntax here https://docs.npmjs.com/files/package.json#engines

Though I'm not really sure why we're specifying this as newer versions of node don't break it and pretty unlikely we'd use a lower version